sanity.json

```diff
@@ -5,7 +5,7 @@
   },
   "api": {
     "projectId": "81pocpw8",
-    "dataset": "production"
+    "dataset": "gro-62-community-ambassadors"
```

Comment:
This will be changed back to "production" before merge; I simply could not figure out a way to get it to use the custom dataset otherwise 😓